scipy/fftpack/logarithmic.py

```diff
@@ -0,0 +1,256 @@
+
+        \tilde{A} = \sqrt{2/\pi} \int_0^\infty A(r) \cos(kr) dr
+
+
+    by making the substitutions
+
+    .. math::
+
+        A(r) = a(r) r^{\mu-1/2},
+
+        \tilde{A}(k) = \tilde{a}(k) k^{-\mu -1/2}
+
+    and applying a biased Hankel transform to a(r).
+
+    Parameters
+    ----------
+    x : array_like, real-valued
+        Array A(r) to transform: a(j) is A(r_j) at r_j = r_c exp[(j-jc) dlnr],
+        where jc = (n+1)/2 = central index of array.
+
+    dlogr : float, optional
+        Separation between input-points (log10); may be positive or negative.
+        Default is 0.01.
+
+    mu : string, optional; {'sine', 'cosine'}
+        Index of J_mu in Hankel transform: 0.5 for a sine transform and -0.5
+        for a cosine transform. Default is 'sine' (0.5).
+
+    q : float, optional
+        Exponent of power law bias; q may be any real number, positive or
+        negative.  If in doubt, use q = 0, for which case the Hankel transform
+        is orthogonal, i.e. self-inverse, provided also that, for n even, kr is
+        low-ringing.  Non-zero q may yield better approximations to the
+        continuous Hankel transform for some functions. Default is 0
+        (unbiased).
+
+    kr : float, optional
+        k_c r_c where c is central point of array
+        = k_j r_(n+1-j) = k_(n+1-j) r_j .
+        Normally one would choose kr to be about 1 (default) (or 2, or pi, to
+        taste). Default is 1.
+
+    rk : float, optional
+        r_c/k_c = r_j/k_j (a constant, the same constant for any j); rk is not
+        (necessarily) the same quantity as kr.  rk is used only to multiply the
+        output array by sqrt(rk)^dir, so if you want to do the normalization
+        later, or you don't care about the normalization, you can set rk = 1.
+        Default is 1.
+
+    n : int, optional
+        Defines the length of the Fourier transform.  If `n` is not specified
+        (the default) then ``n = x.shape[axis]``.  If ``n < x.shape[axis]``,
+        `x` is truncated, if ``n > x.shape[axis]``, `x` is zero-padded.
+
+    axis : int, optional
+        The axis along which the transform is applied.  The default is the
+        last axis.
+
+    overwrite_x : bool, optional
+        If set to true, the contents of `x` can be overwritten. Default is
+        False.
+
+    Returns
+    -------
+    y : real ndarray
+        Transformed array Ã(k): a(j) is Ã(k_j) at k_j = k_c exp[(j-jc) dlnr].
+
+    .. versionadded:: 1.0.0
+
+    References
+    ----------
+    .. [1] 'Uncorrelated modes of the non-linear power spectrum', by A. J. S.
+           Hamilton, `Monthly Notices of the Royal Astronomical Society` vol.
+           312, pp. 257-284, http://dx.doy.org/10.1046/j.1365-8711.2000.03071.x
+           (2000). Website of FFTLog: http://casa.colorado.edu/~ajsh/FFTLog.
+
+    Examples
+    --------
+    >>> from scipy.fftpack import fftlog, fftlogargs
+    >>> # Get fftlog-arguments
```

Comment:
Instead of comments you should use normal text (i.e. remove ``>>> # ``) with blank lines above and below.